### ISSUE TITLE
[ZE0] 12 bars after 80% timer charging

### ISSUE THREAD

**Issue by TeamT** [dalathegreat/BatteryUpgrade-UserManual#12]
80 timer charging work as expected but leave you with 12 SOC bars.

**Comment by dalathegreat** [comment]
Yes! I forgot to list this one here, when 80% limiter is used (or batterysaver), the charge bars will always go to 12/12 on the ZE0. This works perfectly on the AZE0 Leaf, but there is some issue with the older Leaf. Investigating this one for sure when I have time!

**Comment by Afo92** [comment]
Isn't it same issue as "[ZE0] Bars are rescaled incorrectly when charge ends (https://github.com/dalathegreat/BatteryUpgrade-UserManual/issues/22) ?

**Comment by dalathegreat** [comment]
Yes it is @Afo92 !
